### ISSUE TITLE
Unexpected receipt id exception

### ISSUE THREAD

**Issue by Sentoki** [centraldesktop/php-stomp#15]
We use ActiveMQ and this library, and with increasing load on ActiveMQ we begin get this exception.

Process №1 send message with receipt aaa, and get receipt bbb, so we get exception.

Same time process №2 send message with receipt bbb and get receipt aaa, and we get exception to.

I post about this [here](http://activemq.2283324.n4.nabble.com/Working-with-stomp-in-synchronous-mode-and-unexpected-receipt-td4710654.html)  and don't get answer, but i think its right assumption.

So do you faced with this problem and do you have any solution?

Maybe accumulation of some numbers receipts can solve this?

**Comment by treyhyde** [comment]
No, we haven't seen this.   We process ~10k message / sec with ActiveMQ and this library.  ActiveMQ can be sensitive to out of order acks or acking messages when auto-ack is on.     I recommend  ['ack' => 'client-individual'];
